server/models/User.js

```diff
@@ -1,4 +1,15 @@
 import mongoose from "mongoose";
+import Company from "../models/Company.js"
+
+const roles = ['user', 'admin']
```

Comment:
Is this line not suppose to be only 'user', because right now every person is also getting the admin role

Reply:
Thats only for the accepted values